miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/SampleStockSingleCellDto.java

```diff
@@ -0,0 +1,40 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import uk.ac.bbsrc.tgac.miso.core.data.SampleStockSingleCell;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonTypeName(value = SampleStockSingleCell.SAMPLE_CLASS_NAME)
+public class SampleStockSingleCellDto extends SampleStockDto {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 7 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)